reader_test.go

```diff
@@ -1313,3 +1313,45 @@ func getOffsets(t *testing.T, config ReaderConfig) offsetFetchResponseV1 {
 
 	return offsets
 }
+
+const (
+	connTO     = 1 * time.Second
+	connTestTO = 2 * connTO
+)
+
+func TestErrorCannotConnect(t *testing.T) {
+	r := NewReader(ReaderConfig{
+		// FIXME
+		Brokers:     []string{"localhost:9093"},
+		Dialer:      &Dialer{Timeout: connTO},
+		MaxAttempts: 1,
+		Topic:       makeTopic(),
+	})
+	ctx, cancel := context.WithTimeout(context.Background(), connTestTO)
+	defer cancel()
+
+	_, err := r.FetchMessage(ctx)
+	if err == nil || ctx.Err() != nil {
+		t.Errorf("Reader.FetchMessage must fail when it cannot " +
+			"connect")
+	}
+}
+
+func TestErrorCannotConnectGroupSubscription(t *testing.T) {
+	r := NewReader(ReaderConfig{
+		// FIXME
+		Brokers:     []string{"localhost:9093"},
```

Comment:
What requires fixing here?  I think, like above, this test will pass b/c it will be unable to SASL authenticate.

Reply:
Same here.